### ISSUE TITLE
popup does not show Arabic alphabet. it shows like 'Ø¹Ø¨Ø¯Ø§ÙØµÙØ¯ Ø§ÙÙØ±Ø´Ù)'

### ISSUE THREAD

**Issue by sallllam** [python-visualization/folium#1431]
**Describe the bug**

**To Reproduce**

```
[ Include a code snippet that produces your bug. Make it standalone, we should be able to run it. ]

[ Include a data sample or link to your data if necessary to run the code ]

```

**Expected behavior**

**Environment (please complete the following information):**
 - Browser [e.g. chrome, firefox]
 - Jupyter Notebook or html files?
 - Python version (check it with `import sys; print(sys.version_info)`)
 - folium version (check it with `import folium; print(folium.__version__)`)
 - branca version (check it with `import branca; print(branca.__version__)`)

**Additional context**

**Possible solutions**
List any solutions you may have come up with.

folium is maintained by volunteers. Can you help making a fix for this issue?

**Comment by jtbaker** [comment]
I believe this is a duplicate of https://github.com/python-visualization/folium/issues/1320

Please uninstall and reinstall folium and branca directly from the master branch to resolve, until the next release is pushed to PyPI, using

```pip uninstall folium branca -y```

```pip install git+https://github.com/python-visualization/folium.git@master```
```pip install git+https://github.com/python-visualization/branca.git@master```

**Comment by sallllam** [comment]
I did but it does not work

**Comment by Conengmo** [comment]
You only posted a title and didn't fill in the issue template. We can't really help you like this.

**Comment by sallllam** [comment]
![image](https://user-images.githubusercontent.com/24571379/104016505-88c6be00-51c7-11eb-93b4-02ddd7b4ed22.png)

It correctly shows English characters. in the other hand, Arabic characters not in correct way.

**Comment by Conengmo** [comment]
RIght, but what code and data did you use to create this? What software versions did you use? Please take a look at your issue description and fill out the questions listed there.

**Comment by sallllam** [comment]
I use pycharm for flask server and below the code
def location():
    #pd.read_csv("d:/bd/alq.csv")

    lat1 = request.args['lat']
    lng1= request.args['lng']
    df = getinformation(lat1, lng1)
    #df['name']=df['name'].apply(lambda x:str.encode(x,'utf-8'))
    print(df[['name']].head())
    folium_map = folium.Map(location=[lat1,lng1], zoom_start=12)
    folium.Circle(
        location=[lat1,lng1],

        radius=5000,
        color='green',
        fill=True,
        fill_color='green'
    ).add_to(folium_map)

    for lat, lng, label,color in zip(df['lat'], df['lng'], df['name'],df['color']):
        #label= base64.b64encode(label.encode('Image'))
        label=  label.encode('utf-8-sig')
        #label = folium.Popup(label.encode ('utf-8-sig'), parse_html=True)

        folium.Marker([lat, lng], radius=10, popup=label, color=color, fill=True, fill_color='black',icon = folium.Icon(color=color),
                            fill_opacity=0.7, parse_html=False).add_to(folium_map)

    folium.Marker([lat1, lng1], radius=10,popup="location for new store", color="green", fill=True, fill_color='green', icon = folium.Icon(color='green'),
                  fill_opacity=0.7, parse_html=False).add_to(folium_map)
    #folium.Circle([lat1, lng1], radiou=150000, color="blue", fill=True, fill_color='blue').add_to(folium_map)
    return folium_map._repr_html_()
the data is in this link: https://github.com/sallllam/copston/blob/main/foliumdf.csv 


libraries:
Flask==1.1.2
folium==0.0.0 after install it from  pip install git+https://github.com/python-visualization/folium.git@master but before it was 0.12
branca==0.4.2 after install it from pip install git+https://github.com/python-visualization/branca.git@master

**Comment by sallllam** [comment]
please help

**Comment by matkoniecz** [comment]
> the data is in this link: https://github.com/sallllam/copston/blob/main/foliumdf.csv

link is dead

> utf-8-sig

Is CSV file in a matching encoding?

**Comment by Conengmo** [comment]
I'm pretty sure this has been fixed with branca v0.4.2.
